go/ct/spc/specification.go

```diff
@@ -1839,6 +1956,75 @@ func logOp(n int) []Rule {
 	return rules
 }
 
+func selfDestructEffect(s *st.State, destinationColdCost, dynamicGas, refoundGas vm.Gas) {
+	// Behavior pre cancun: the current account is registered to be destroyed, and will be at the end of the current
+	// transaction. The transfer of the current balance to the given account cannot fail. In particular,
+	// the destination account code (if any) is not executed, or, if the account does not exist, the
+	// balance is still added to the given address.
+
+	// account to send the current balance to
+	destinationAccount := s.Stack.Pop().Bytes20be()
+	currentAccount := s.CallContext.AccountAddress
+	CurrentBalance := s.Accounts.GetBalance(currentAccount)
+
+	// If a positive balance is sent to an empty account, the dynamic gas is 25000. Otherwise there is
```

Comment:
The documentation does not match the implementation. The 25000 are only true if the `dynamicGas` is set accordingly.